aptos-move/framework/aptos-stdlib/sources/simple_map.spec.move

```diff
@@ -56,8 +57,9 @@ spec aptos_std::simple_map {
     }
 
     // Specification functions for tables
+    spec native fun spec_len<K, V>(t: SimpleMap<K, V>): num;
     spec native fun spec_contains_key<K, V>(t: SimpleMap<K, V>, k: K): bool;
-    spec native fun spec_add<K, V>(t: SimpleMap<K, V>, k: K, v: V): SimpleMap<K, V>;
+    spec native fun spec_set<K, V>(t: SimpleMap<K, V>, k: K, v: V): SimpleMap<K, V>;
```

Comment:
Why change this?

Reply:
The name has been changed from `spec_add` to `spec_set` in Prover because the latter better represents its meaning. This change is to reflect it.